ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -160,6 +161,33 @@
                   "run_module"
             ]
         },
+        "run-job-template-action": {
+            "type": "object",
+            "properties": {
+                "run_playbook": {
```

Comment:
Is this an error? I think it must be `run_job_template`

Reply:
typo. thanks.